README.md

```diff
@@ -1,39 +1,106 @@
+<div align="center">
+    <a href="https://github.com/CrayLabs/SmartSim"><img src="https://raw.githubusercontent.com/CrayLabs/SmartSim/master/doc/images/SmartSim_Large.png" width="90%"><img></a>
+    <br />
+    <br />
+    <div display="inline-block">
+        <a href="https://github.com/CrayLabs/SmartSim-Scaling"><b>Home</b></a>&nbsp;&nbsp;&nbsp;
+        <a href="https://www.craylabs.org/docs/installation_instructions/basic.html"><b>Install</b></a>&nbsp;&nbsp;&nbsp;
+        <a href="https://www.craylabs.org/docs/overview.html"><b>Documentation</b></a>&nbsp;&nbsp;&nbsp;
+        <a href="https://join.slack.com/t/craylabs/shared_invite/zt-nw3ag5z5-5PS4tIXBfufu1bIvvr71UA"><b>Slack</b></a>&nbsp;&nbsp;&nbsp;
+        <a href="https://github.com/CrayLabs"><b>Cray Labs</b></a>&nbsp;&nbsp;&nbsp;
+    </div>
+    <br />
+    <br />
+</div>
+----------
+
 # SmartSim Scaling
 
 This repository holds all of the scripts and materials for testing
-the scaling of SmartSim and the SmartRedis clients.
+the scaling of SmartSim and SmartRedis clients.
+
+The scalability tests mimic an HPC workload by making calls to SmartSim 
+and SmartRedis infrastructure to complete in parallel highly complex, data-intensive 
+tasks that are spread across compute resources. 
+These applications are used to test the performance of SmartSim and 
+SmartRedis across various system types.
+
+## Scalability Tests Supported
+
+The SmartSim-Scaling repo offers three scalability tests with 
+six total versions detailed below:
+
+#### `Inference Tests`
+
+| Inference Type | Client | Parallelization |
+| :--- | --- | --- |
+| Standard | C++ AND Fortran | MPI |
+| Colocated | C++ AND Fortran | MPI |
+
+#### `Throughput Tests`
+
+| Throughput Type | Client | Parallelization |
+| :--- | --- | --- |
+| Standard | C++ | MPI |
+| Colocated | C++ | MPI |
+
+#### `Data Aggregation Tests`
+
+| Data Aggregation Type | Client | Parallelization |
+| :--- | --- | --- |
+| Standard | C++ | MPI |
+| Standard | Python | MPI |
+| Standard | Python | File System |
 
+## Colocated vs Standard Deployement
 
-## Scaling Tests
+The scaling repo offers two types of Orchestrator deployment: Standard and Colocated.
 
-There are two types of scaling tests in the repository.
+> Note that the Orchestrator is a SmartSim term for a Redis or KeyDB database with a RedisAI module built into it with the ML runtimes.
```

Comment:
Delete Note that. Also, I think the Orchestrator usually includes SmartRedis

Reply:
Could you clarify what you mean by the orch includes smartredis?

Reply:
You are equating the Orchestrator with Redis/KeyDB. But the orchestrator for our purposes is SmartRedis wrapped around Redis (or KeyDB)

Reply:
the orch is the db and the smartredis client software around it